app/src/main/java/org/freedombox/freedombox/views/model/ConfigModel.kt

```diff
@@ -0,0 +1,41 @@
+package org.freedombox.freedombox.views.model
+
+import android.os.Parcel
+import android.os.Parcelable
+
+data class ConfigModel( var boxName: String,
```

Comment:
Try using this:
https://github.com/Kotlin/KEEP/blob/master/proposals/extensions/android-parcelable.md

Reply:
```
Type: Design proposal
Status: Under consideration
Prototype: In progress
```